pom.xml

```diff
@@ -57,6 +57,11 @@
             <artifactId>spring-boot-configuration-processor</artifactId>
         </dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
```

Comment:
That's a big dependency that may break projects of other users.
@reitzmichnicht Do you have an opinion since you optimized the dependencies?